cmd/vulcan-tracker/main.go

```diff
@@ -68,6 +80,7 @@ func main() {
 	e.POST("/:team_id/tickets/:id/fix", a.FixTicket)
```

Comment:
These functionalities had already been developed before the decision was taken to do manual ticket creation.
If you consider it necessary, I can remove the endpoints so that they are not invoked. In any case, I would leave the implementation and remove only these lines so that they are not exposed.
WDYT?

Reply:
I agree it makes sense.